### PR TITLE
New package: madshi.eac3to version 3.36

### PR DESCRIPTION
**Pull request by Dragon1573**
## What's changed

- Resolve https://github.com/microsoft/winget-pkgs/issues/119634

## Notes

It seems a "loose executable" zip compressed file. Succcessfully installed but the main executable `eac3to.exe` cannot launch in terminal (it's a CLI application).

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122569)

### CODE REVIEW

**Comment by Dragon1573** [comment]
Installer URL valid on local device and Windows Sandbox. No malicious code are detected by QIANXIN Tianqing (奇安信天擎). 

Need help.

**Comment by stephengillie** [comment]
Automated Validation ended with:
> Installer Verification Analysis Context Information: 
Installer http://madshi.net/eac3to.zip
Scan request id: 'a3a9fd38-86e9-4ed3-83d1-4fc916de8e33'
Error information: BlockingDetectionFound

The download is blocked, possibly to prevent automated downloads. Unfortunately, this error can't be easily waivered or worked around.

**Comment by Dragon1573** [comment]
Uploaded the zip file to VirusTotal and 2 vendors flagged this file as malicious. Report [here](https://www.virustotal.com/gui/file/e7977cf3e87310619986b3d1d30385f66b297b189594ead3d6518b0673c05905/detection).

Does validation pipeline use a special user-agent to flag itself a WinGet automation bot? Or it was detected via other labels or tags? Can we simulate some identity of the pipeline locally and try if we can download the file?

File: manifests/m/madshi/eac3to/3.36/madshi.eac3to.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Publisher: madshi.net
+PublisherUrl: http://madshi.net/
+Author: madshi.net
+PackageName: eac3to
+PackageUrl: https://forum.doom9.org/showthread.php?t=125966
+License: Proprietary
+ShortDescription: An audio conversion tool.
+InstallationNotes: |
+  This is a commandline "loose executable" file, but currently can't be launched via terminal.
+  Please navigate to folder "%LOCALAPPDATA%/Microsoft/WinGet/Links" and launch manually.
```
Comment (Trenly) [inline manifests/m/madshi/eac3to/3.36/madshi.eac3to.locale.en-US.yaml:16]:
```suggestion
  Please navigate to the install folder and launch manually.
```
Not all installs support the links directory, and users can change where the directory is created. Please avoid putting paths into install notes

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This package installs normally, and runs normally when launched from its package location. But the symlink doesn't provide any output to PowerShell - the only way to get output is to run from Explorer, and then it opens a new window, streams text instructions, then closes the window. And since this happens very quickly, the only way to pause for a screenshot it was to quickly click and drag across the console window: 
image
- Despite being launched by the symlink, the window shows the main executable's path. 

I'm guessing this is because the symlink doesn't have access to library files that the main does. What are your thoughts?

**Comment by Dragon1573** [comment]
@stephengillie Yep. I also meet some package with this behaviour. Most of them are "loose executable".

Despite from adding a special directory to the `$env:PATH` and create symlinks for executable, is it possible to add the directory to `PATH` directly where the executable exists?

Maybe this performance is only acceptable after the "pre/post installation script" feature is available?

**Comment by stephengillie** [comment]
To my knowledge, the WinGet package manager doesn't currently include this functionality. A feature request could be a start to getting it.

**Comment by Dragon1573** [comment]
This pull request is block because of an unavailable feature mentioned below. Convert this pull request to draft for waiting the feature and prevent other contributors from wasting time on this work.

- Related https://github.com/microsoft/winget-pkgs/pull/122569#issuecomment-1776409256

**Comment by Dragon1573** [comment]
Seems no more focus on this pull request. Close without merge.

**Pull request closed without merging**